ext/pcntl/run-tests.php

```diff
@@ -0,0 +1,2843 @@
+#!/usr/bin/env php
```

Comment:
This file is not supposed to be part of the PR I can rebase and make the history clean ?